app/notifications/validators.py

```diff
@@ -144,10 +144,19 @@ def check_if_service_can_send_to_number(service, number):
         return international_phone_info
 
 
-def check_content_char_count(template_with_content):
+def check_content_character_count(template_with_content):
     if template_with_content.is_message_too_long():
-        message = f"Text messages cannot be longer than {SMS_CHAR_COUNT_LIMIT} characters. " \
-                  f"Your message is {template_with_content.content_count_without_prefix} characters"
+        message = "Your message is too long."
+        if template_with_content.template_type == SMS_TYPE:
+            message = (
+                f"Text messages cannot be longer than {SMS_CHAR_COUNT_LIMIT} characters. "
+                f"Your message is {template_with_content.content_count_without_prefix} characters."
```

Comment:
You've added a full stop on the end here. Is that intentional? I don't know if that would mean it no longer matches our API documentation? Or if we should have one to start with?

Reply:
One for @karlchillmaid I guess, I changed it because it felt weird that we had no gull stop and then we had a capital letter 🤷

Reply:
I think the content looks good. Skimming through the other error messages in this file, it looks like we might be a bit inconsistent about full stops in API messages. Something to fix in the future?